packages/component-library/draft/Kaizen/DatePicker/DatePicker.tsx

```diff
@@ -0,0 +1,91 @@
+import { Popover, TextField } from "@cultureamp/kaizen-component-library/draft"
+import { Moment } from "moment"
+import React, { useCallback, useEffect, useMemo, useRef, useState } from "react"
+import Calendar from "./primitives/Calendar"
+
+const dateIcon = require("@cultureamp/kaizen-component-library/icons/date-start.icon.svg")
+  .default
+const dateRangeIcon = require("@cultureamp/kaizen-component-library/icons/date-range.icon.svg")
+  .default
+
+const styles = require("./styles.scss")
+
+export type DatePickerProps = {
+  id: string
```

Comment:
Does this need an `id` to function, and if it does, could this be an optional prop?

Reply:
Ah, my mistake. Looks like this is required for TextField. Disregard!